recipes/Fundamental Lentil Soup.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Paul Campbell
+residence: Duvall, WA, USA
+category: entree
+---
+
+# Fundamental Lentil Soup
+
+> It's important to nail the fundamentals in any project -- even dinner.
+
+## Ingredients
+
+2 cups green lentils, rinsed
+1/2 onion, chopped medium
+3 carrots, peeled and chopped
+3-4 stalks celery, chopped
+4 cloves garlic, minced or crushed
+2 tsp dry thyme
+1 tbsp dry parsley
+1 bay leaf
+1 tbsp olive (or other) oil
+4 cups vegetable broth
+salt
+pepper
+
+## Directions
+
+1. in large sauce pan, saute onions in olive oil on medium heat until clear (5-7 minutes)
```

Comment:
Though not complete sentences, I think it would look nice to see the first letters capitalized.